src/boxes/mp4_box.rs

```diff
@@ -31,6 +35,9 @@ pub trait Mp4Box: Downcast {
 
     /// The box level
     fn level(&self) -> u8;
+
+    /// Print offsets of stco boxes
+    fn offsets(&self) -> Option<Vec<u64>>;
```

Comment:
I named the method a "offset()" just because I didn't think of the right name for the method. The original intention to include this method is to parse additional data further down the atom, apart from atom name and atom size which we already have as part of the logic. For example, ftyp may have additional metadata such as majorBrand, minorVersion, so forth. Similarly, vmhd and smhd have further metadata such as opcolor_r, graphicsmode, and so forth. Similarly, ChunkOffsetTable is one of the metadata in the stco atom. just like how other atoms will have a metadata but in different form. 

Now one might argue that we might not require this, but wouldn't it be good to be prepared if we might need this metadata for example for redaction or conversion process?

Reply:
I'm not saying we do not require such methods, they should be in their right place. The `Mp4Box` trait is for defining the common functions for all other modules. If you need more functions in a module, you should have them in the module impl block.